### ISSUE TITLE
Training data

### ISSUE THREAD

**Issue by tkisss** [JiaqiLiZju/Nvwa#6]
Hi Jiaqi,

Since you obtained the sequence of each gene from the reference genome based on coordinates, the sequences of the same gene in different cells are completely identical. However, their gene expression may vary. How does NVWA handle this situation?

Thanks！

**Comment by HelloWorldLTY** [comment]
If my understand is correct, Nvwa treated input data as sequence information of gene, and the output is that gene's expression across all the cells. Therefore, Nvwa is predicting the gene expression in all the cells, and that is why they split training and testing datasets by genes.

**Comment by JiaqiLi1024** [comment]
> If my understand is correct, Nvwa treated input data as sequence information of gene, and the output is that gene's expression across all the cells. Therefore, Nvwa is predicting the gene expression in all the cells, and that is why they split training and testing datasets by genes.

Yes, exactly, thanks for your reply!

**Comment by JiaqiLi1024** [comment]
Not so Formally, for a specific Gene -g, a specific Cell -c, 
`Expression prediction of {g, c} = Linear_func_of_Cell_c( CNN( Sequence_of_Gene_g) )`;

The same gene will have identical embedding after `CNN`, but each cell will have its specific parameters in `Linear_func_of_Cell_c `, so the Expression prediction will be cell-specific.

**Comment by tkisss** [comment]
Do you mean that NVWA needs to train a model for each gene in every cell, for example, model{g,c} for gene g in cell c, rather than using a general model?

**Comment by JiaqiLi1024** [comment]
Nope, it's a general model. I just informally formulate the parameters inside the model architecture.

**Comment by tkisss** [comment]
I see, thanks!
